docs/ABOUT.md

```diff
@@ -0,0 +1,3 @@
+# About
+
+Powershell is a command-line shell and an object oriented scripting language. Powershell is to windows as Bash is to Linux. Powershell is built into nearly all windows distributions, and is also able to run on OSX and linux. [Additional Information](https://en.wikipedia.org/wiki/PowerShell)
```

Comment:
It looks like the official spelling is PowerShell (as opposed to Powershell). I think it's worth being consistent about that.